modules/digitaluniverse/rendering/renderablebillboardscloud.cpp

```diff
@@ -644,18 +643,17 @@ void RenderableBillboardsCloud::initializeGL() {
     _program = DigitalUniverseModule::ProgramObjectManager.request(
         ProgramObjectName,
         []() -> std::unique_ptr<ghoul::opengl::ProgramObject> {
-            return global::renderEngine.buildRenderProgram(
-                ProgramObjectName,
-                absPath("${MODULE_DIGITALUNIVERSE}/shaders/billboard_vs.glsl"),
-                absPath("${MODULE_DIGITALUNIVERSE}/shaders/billboard_fs.glsl"),
-                absPath("${MODULE_DIGITALUNIVERSE}/shaders/billboard_gs.glsl")
-            );
-        }
+        return global::renderEngine.buildRenderProgram(
+            ProgramObjectName,
+            absPath("${MODULE_DIGITALUNIVERSE}/shaders/billboard_vs.glsl"),
+            absPath("${MODULE_DIGITALUNIVERSE}/shaders/billboard_fs.glsl"),
+            absPath("${MODULE_DIGITALUNIVERSE}/shaders/billboard_gs.glsl")
+        );
+    }
     );
 
     _renderToPolygonProgram = DigitalUniverseModule::ProgramObjectManager.request(
-        RenderToPolygonProgram,
-        []() -> std::unique_ptr<ghoul::opengl::ProgramObject> {
+        RenderToPolygonProgram, []() -> std::unique_ptr<ghoul::opengl::ProgramObject> {
```

Comment:
was better the way it was before.. and no real need to specify the return type explicitly